pf/tests/schemashim_test.go

```diff
@@ -74,12 +85,48 @@ func TestSchemaShimRepresentations(t *testing.T) {
       }
     }
   }
+}`),
+			autogold.Expect(`{
```

Comment:
This expansion of the test is optional but it helped me sanity-check the schemas generated for each case by PF.